resources/js/processes-catalogue/components/ProcessTab.vue

```diff
@@ -5,29 +5,359 @@
         title="My Requests"
         active
       >
-        <default-tab
-          :altText="$t('No Image')"
-          :titleText="$t('You have made no requests of this process.')"
-          :descriptionText="$t('All your requests will be shown here')"
-        />
+        <div v-if="showTabRequests">
+          <filter-table
+            :headers="tableHeadersRequests"
+            :data="dataRequests"
+            @table-row-click="(row) => handleRowClick('request', row)"
+          />
+          <pagination-table
+            :meta="dataRequests.meta"
+            @page-change="changePage"
+          />
+        </div>
+        <div v-else>
+          <default-tab
+            :altText="$t('No Image')"
+            :titleText="$t('You have made no requests of this process.')"
+            :descriptionText="$t('All your requests will be shown here')"
+          />
+        </div>
       </b-tab>
       <b-tab title="My Tasks">
-        <default-tab
-          :altText="$t('No Image')"
-          :titleText="$t('You have no tasks from this process')"
-          :descriptionText="
-            $t('All your tasks related to this process will be shown here.')
-          "
-        />
+        <div v-if="showTabTasks">
+          <filter-table
+            :headers="tableHeadersTasks"
+            :data="dataTasks"
+            @table-row-click="(row) => handleRowClick('task', row)"
```

Comment:
use openTasks instead of handleRowClick

Reply:
method handleRowClick is necessary, using openTasks generates error